docs/ru/ddrone.md

```diff
@@ -0,0 +1,37 @@
+# Граффити коптер D-drone
+
+<iframe width="560" height="315" src="https://youtu.be/ErtioCj5iMw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Введение
+Люди стремятся научить искусственный интеллект всему, что могут делать сами. Рисовать нас учат с детства. И почему бы не научить дрона рисовать? На данный момент коптеры и граффити набирают свою популярность. Поэтому мы решили совместить их.
+
+## Модели и сборка
+<p align = "center">
```

Comment:
Please, don't use `p` tags in the article, we have a `center` class for centering the image.